cmd/libs/go2idl/client-gen/autoimport/target.go

```diff
@@ -0,0 +1,5 @@
+package autoimport
```

Comment:
You have the right technique, but you should not need to make this package to get it to work. Just use "fmt" and "Errorf" directly?